Formula/manticoresearch.rb

```diff
@@ -0,0 +1,85 @@
+class Manticoresearch < Formula
+  desc "Open source text search engine"
+  homepage "https://www.manticoresearch.com"
+  url "https://github.com/manticoresoftware/manticoresearch/releases/download/3.1.0/manticore-3.1.0-190716-445e806-release.tar.gz"
+  version "3.1.0"
+  sha256 "b8eba31eea5f6f5cf7cb7986c94cb40904f96da99e178c724e7f0081de02388e"
+  head "https://github.com/manticoresoftware/manticoresearch.git"
+
+  depends_on "cmake" => :build
+  depends_on "icu4c" => :build
+  depends_on "libpq" => :build
+  depends_on "mysql@5.7" => :build
```

Comment:
We do not want to ship new formulas that depend on old versions of build tools or libraries. Can this be built with latest `mysql`?